components/service-operator/Makefile

```diff
@@ -36,9 +36,9 @@ deploy: manifests
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-	cat config/crd/bases/*.yaml > ../../charts/gsp-cluster/templates/02-gsp-system/service-operator/crd.yaml
-	cp config/rbac/role.yaml ../../charts/gsp-cluster/templates/02-gsp-system/service-operator/role.yaml
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./..." output:crd:stdout > ../../charts/gsp-cluster/templates/02-gsp-system/service-operator/crd.yaml
```

Comment:
Is the output of this predictable? It would be nice to not have a diff like this one every time CRDs are added to this project...

Reply:
Maybe splitting the CRD files out into separate ones would be better.